### DISCUSSION TITLE
Change ports in docker-compose.yml

### DISCUSSION THREAD

**Discussion by PEQSPC** [globaleaks/globaleaks-whistleblowing-software#3932] [General]
Can i change ports in docker-compose.yml because is causing conflict in my alma linux?
![2024-01-05 09_53_53-Podman containers - admin@globaleaks — Mozilla Firefox](https://github.com/globaleaks/GlobaLeaks/assets/124091719/f040fe30-8b97-4eb5-8cac-bd877c712026)
when a run podman-compose up -d

**Comment by evilaliv3** [comment]
@PEQSPC : i actually do not think the problem is on the ports.

@rglauco by any chance do you have any advice?

**Comment by rglauco** [comment]
@PEQSPC I tried on Almalinux 9 with default docker-compose.yml, no problem at all when changing exposed ports.
The error is related to something else, could you please paste the container logs?

**Reply by PEQSPC** [reply]
the logs of my container as only that message :
![imagem](https://github.com/globaleaks/GlobaLeaks/assets/124091719/b71ac7b4-4c4b-410a-87b6-6bfb546bbc62)

**Reply by PEQSPC** [reply]
![imagem](https://github.com/globaleaks/GlobaLeaks/assets/124091719/fc5839dd-a3f9-438b-9cb9-d2c1161dbaaf)

**Comment by PEQSPC** [comment]
What steps did you take to install? @rglauco

**Comment by rglauco** [comment]
With the default Dockerfile and docker-compose.yml, i've only launched `podman-compose up -d`.
I tried as root and as regular user with custom ports in both setups with podman version 4.6.1.

Your error seems to be related inside the container, do you use a custom Dockerfile?

**Reply by PEQSPC** [reply]
No,i  use the default dockerfile in the this git repository

**Reply by PEQSPC** [reply]
i will try to create a ubuntu server container and then install with this InstallationGuide https://docs.globaleaks.org/en/main/setup/InstallationGuide.html

**Comment by rglauco** [comment]
Could you give `podman-compose down -v` and delete the globaleaks image?
Then give `podman-compose up` and copy-paste the logs (as text file, not image, for better readability)?

**Reply by PEQSPC** [reply]
[logsglobal.txt](https://github.com/globaleaks/GlobaLeaks/files/13859289/logsglobal.txt)

**Comment by PEQSPC** [comment]
its working now i think

**Comment by rglauco** [comment]
Yes, it seems so. Now you can try to change exposing host ports.

**Reply by PEQSPC** [reply]
how i backup the container with all this data ?

**Comment by rglauco** [comment]
I believe with `podman-commit --include-volumes`.
Maybe it's better to save only data volume, i'll let you find a proper way to do so: with `podman volume inspect docker_globaleaks` you find the volume location (i guess `/home/admin/.local/share/containers/storage/volumes/docker_globaleaks/_data`) there you find the data.

**Reply by PEQSPC** [reply]
ok, thank you very much for your help
